sandstone/lib/handlers/broadcast.py

```diff
@@ -1,19 +1,47 @@
 import tornado.ioloop
 import tornado.web
 import tornado.websocket
+import tornado.escape
 import json
 from pydispatch import dispatcher
 
+
+class BroadcastManager(object):
```

Comment:
Since `broadcast.py` does not just contain a handler and the system is likely to expand, this file should be converted to a submodule within `lib`, with files `manager.py` and `handlers.py`.